geppetto.js/geppetto-client/js/components/interface/LoadingSpinner/LoadingSpinner.js

```diff
@@ -0,0 +1,43 @@
+
+import React from 'react';
+
+import Loader from '@metacell/geppetto-meta-ui/loader/Loader';
+
+export default class LoadingSpinner extends React.Component{
+
+  /**
+   * Legacy way to hide the spinner
+   */
+  hideSpinner (){
```

Comment:
inconsistent formatting in various files, from different use of whitespaces (e.g. braces) to too many empty lines
eslint rules don't complain yet (but soon -> #63) we can leave it, don't want to create extra effort

Reply:
The formatting depends on how the ide interprets the rules. I'd leave all formatting issues after more code pruning